packages/@vue/babel-preset-app/index.js

```diff
@@ -14,8 +14,8 @@ const defaultPolyfills = [
 ]
 
 function getPolyfills (targets, includes, { ignoreBrowserslistConfig, configPath }) {
-  const { isPluginRequired } = require('@babel/preset-env')
-  const builtInsList = require('@babel/preset-env/data/built-ins.json')
+  const { isPluginRequired } = require('@babel/preset-env/lib/filter-items')
+  const builtInsList = require('@babel/preset-env/data/corejs2-built-ins.json')
```

Comment:
Hi, Babel maintainer here 👋

We just released v7.4.1, which should fix the issue. The `builtInsList` change is correct, but I'd ask you to revert line 17 (it's better not to rely on internal files, when possible). You'll need to update the `preset-env` version in package.json.

Reply:
Hi! Thank you for the quick reply. I noticed that you also aliased `cosejs2` so both my changes are unnecessary right now. Perhaps we still need to change the path of `built-ins.json` if the alias is removed in the future.